python/lsst/ip/isr/isrMock.py

```diff
@@ -532,7 +546,7 @@ def makeImage(self):
             return exposure
 
     # afw primatives to construct the image structure
-    def getCamera(self):
+    def getCamera(self, isForAssembly=False):
         """Construct a test camera object.
 
```

Comment:
This needs a parameter docstring to explain what is going on here.  As I understand it, this should be set to True when (a) the camera is lsst like *and* (b) the camera is being used in the `getExposure()` method where it will then be assembled into the final form and rearranged.  In all other cases this should be left False so that the camera retrieved here matches the detector attached to the exposure.